docs/source/quickstart.mdx

```diff
@@ -283,8 +283,8 @@ type Department {
 scalar join__FieldSet
 
 enum join__Graph {
-  PRODUCTS @join__graph(name: "products" url: "https://7bssbnldib.execute-api.us-east-1.amazonaws.com/Prod/graphql")
-  REVIEWS @join__graph(name: "reviews" url: "https://w0jtezo2pa.execute-api.us-east-1.amazonaws.com/Prod/graphql")
```

Comment:
ooh - umm, i wonder if these URLs will actually be updated in the schema.. likely not? lemme check

Reply:
ohh yeah this is right! i forgot that these come from `routing_url`. we're good 😄